spec/features/events/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "Events" do
```

Comment:
This is vague for a top-level description. Perhaps this is really testing "Viewing the events list" (same for the edit_spec)

Reply:
When reading the rspec documentation format, it could be nice to have it be more explicit.

Reply:
Rubocop complains if the describe doesn't match the object under test.
![image](https://user-images.githubusercontent.com/8124558/128642430-4bb79a36-56b4-4b9a-ab8a-a4ab1381a83d.png)
Anything other than `Profile` isn't accepted. This might be a bigger issue. For now, Profile is compliant with other feature specs in the repo, and passes Rubocop.

Also, Rubocop doesn't like feature either. This isn't the right PR to rewrite rubocop rules. Let's open a separate issue to rename the feature specs.

Reply:
👎  for rubocop. But, very big 👍  for putting this off to another issue.

Reply:
Made an issue! #34